frontend/wallet/src/render/components/Button.re

```diff
@@ -27,63 +30,72 @@ module Styles = {
       ]),
     ]);
 
-  let blue =
-    merge([
-      base,
-      style([
-        backgroundColor(Theme.Colors.marineAlpha(0.1)),
-        color(Theme.Colors.marineAlpha(1.)),
-        hover([backgroundColor(Theme.Colors.marine), color(white)]),
-        focus([backgroundColor(Theme.Colors.marine), color(white)]),
-        active([backgroundColor(Theme.Colors.marine), color(white)]),
-      ]),
-    ]);
+  let buttonLink = style([textDecoration(`none), color(white)]);
```

Comment:
instead of prefixing the varaibles withthe name `button`, can you wrap these in `module Button = { }`?

Reply:
do you mean inside of module Styles = { } ?

Reply:
or its own outside of Styles?

Reply:
or instead of Styles completely haha